ios/Podfile

```
@@ -48,7 +49,25 @@ end
+  def scrub_cp_filelists_from_target(target)
+    target.build_phases.each do |phase|
+      next unless phase.respond_to?(:name)
+      next unless phase.name&.include?('[CP]')
+      begin
+        phase.input_paths = [] if phase.respond_to?(:input_paths=)
+        phase.output_paths = [] if phase.respond_to?(:output_paths=)
+      rescue => e
+        puts "⚠️  Skipping scrub for phase #{phase.name}: #{e}"
+      end
+    end
+  end
```

RuboCop (1.76.1)

[convention] Cyclomatic complexity for `scrub_cp_filelists_from_target` is too high. [8/7]

(Metrics/CyclomaticComplexity)